components/amount-buttons.jsx

```diff
@@ -16,17 +16,17 @@ var AmountButton = React.createClass({
     if (this.props.value === this.props.amount) {
       checked = true;
     }
-
     return (
       <div className="third">
         <input onChange={this.props.onChange} checked={checked} className="amount-radio" type="radio" name="donation_amount" value={this.props.value} id={"amount-" + this.props.value}/>
         <label htmlFor={"amount-" + this.props.value} className="amount-button large-label-size">
+          { this.props.currencyCode && this.props.value ?
           <FormattedNumber
             minimumFractionDigits={0}
             value={this.props.value}
             style="currency"
-            currency={this.props.currencyCode || "usd"}
-          />
+            currency={this.props.currencyCode}
+          /> : <span>&nbsp;</span> }
```

Comment:
I expected the '' to do that too, then I tried ' ' which I was also annoyed didn't work. Then I tried '&nbsp;' which displayed the text. lol, so adding it to a span not as text fixed it.